packages/bako-safe/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@fuel-connectors/bako-safe",
```

Comment:
It's missing to add a changeset for this package.
Just run `pnpm changeset` and follow the instructions.